fenix/app/src/main/res/color/toggle_positive_button_text_color.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_enabled="false" android:alpha="0.50" android:color="?attr/textPrimary" />
+    <item android:color="?attr/textOnColorPrimary"/>
+</selector>
```

Comment:
Rather than introduce a new color here, I think I would prefer to just change the alpha of the text color through the code in `SaveSearchEngineFragment` where we are already updating the `saveButton.isEnabled` property

Reply:
Would it be a better way though? Here we just allow system to handle it, as we manage the enabled state. The alternative would be something like

```
 val textColor = binding.saveButton.currentTextColor
 val textColorWithAlpha = ColorUtils.setAlphaComponent(textColor, if (bothFieldsHaveInput) 255 else 127)
 binding.saveButton.setTextColor(textColorWithAlpha)
```
to be triggered on create and on text changed?

Reply:
I personally prefer to read code rather than need to search through XML files, but I don't care enough to block on this. FWIW, I think the above could also be

```
            if (bothFieldsHaveInput) {
                binding.saveButton.isEnabled = true
                binding.saveButton.alpha = 1f
            } else {
                binding.saveButton.isEnabled = false
                binding.saveButton.alpha = 0.5f
            }
```

Reply:
Oh, I see what you mean. We could even move it into two lines.
```
binding.saveButton.isEnabled = bothFieldsHaveInput
binding.saveButton.alpha = if (bothFieldsHaveInput) 1f else 0.5f
```

But this file doesn't just solve the `alpha` problem, it also adds a colour state for the button to be [white](https://www.figma.com/file/WzMZduDsU0dJUj8P3gqiPf/Search-engine-settings?type=design&node-id=262-27194&t=rUBEH49xfwNHzIHm-4) in enabled mode. 

Which made me realize that what we actually need is another state list color. Added `text_on_color_state_list_text_color.xml`